README.md

```diff
@@ -13,6 +13,19 @@ Download and install the following tools:
 * [checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html)
 * [Google java format](https://github.com/google/google-java-format)
 
+* In order to setup this project it's necessary:
```

Comment:
You need to make sure the migrations ran in python side, so you can point to the section in the python repo that runs the migrations

Reply:
Done.